engine/kafka/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaConfig.scala

```diff
@@ -22,4 +23,6 @@ object KafkaConfig {
     config.as[KafkaConfig](path)
   }
 
+  def parseProcessObjectDependencies(processObjectDependencies: ProcessObjectDependencies): KafkaConfig =
```

Comment:
why we fix "kafka" here?

Reply:
Because it's default kafka namespace from processObjectDependencies.config